HedgeModManager/Resources/Codesv2/SonicFrontiers.hmm

```diff
@@ -1009,6 +1009,98 @@ WARNING: this will almost definitely cause performance issues, especially if you
     }
 }
 
+Code "Additive Springs on Boost" in "Gameplay" by "WasifBoomz" does "Keeps your velocity upon hitting a vertical spring while boosting"
```

Comment:
Rename to "Additive Spring Trajectory with Boost".

Add a period to the end of the description.